js/templates/pageNav.html

```diff
@@ -42,43 +38,45 @@
             style="background-image: url('../imgs/defaultUser.png')"
           <% } %>
           ></a>
-          <nav class="navList listBox clrBr clrP shad1 foldDown js-navList js-navPopMenu">
-            <div class="listGroup clrS clrBr">
-              <a class="listLink clrS clrBr">Current Node <i
-                class="ion-arrow-right-b floR"></i> </a>
+          <nav class="navList listBox clrS clrSh1 foldDown js-navList js-navPopMenu">
+            <div class="listGroup clrP clrBr clrSh2">
+              <a class="listItem">
+                <span>Current Node</span>
+                <span><i class="ion-arrow-right-b floR"></i></span>
+              </a>
             </div>
-            <div class="listGroup clrS clrBr">
-              <a class="listLink">
+            <div class="listGroup clrP clrBr clrSh2">
+              <a class="listItem" href="#ownPage">
                 <span>My Page</span><span class="clrT2">Cltrl + ?</span>
               </a>
-              <a class="listLink">
+              <a class="listItem"><!--TODO add js action for Page Customization-->
```

Comment:
Let's circle back and talk about this one. I know it may sound like I'm talking about both sides of my mouth here, but I don't think we want a JS action for this, otherwise we'd have to have both the page-nav and router manage opening the user page, which IMO is not worth it at all.

If the idea is just to keep customize out of history, I think from the pageNav we could probably link to 'ownPage/customize' and just have the user route re-route to '<guid>/about' while giving a 'customize' flag to the userPage view.

For clicks of the Customize CTA within the userPage, we don't even need to go through the router. There we could just go into the customize flow via JS since we wouldn't want to lose other page state (like the tab they're on). That's how it's done in v1 and it seems to work.

Sorry for any confusion.